### PR TITLE
[WIP] Release candidate 52.0.0

### PR DESCRIPTION
**Pull request by romoore**

### CODE REVIEW

**Comment by romoore** [comment]
Looks like all internal tests are now passing with a few expected changes (Text Fields, Chips). Rerunning regression tests one final time before merging to stable.

File: CHANGELOG.md [hunk 1]
```diff
@@ -70,11 +70,11 @@
 
 * [Simplify color themer (#3116)](https://github.com/material-components/material-components-ios/commit/f0368d43838c35ae463afad3a5f75cead61753b5) (Robert Moore)
 
-### ButtonBar
+#r## ButtonBar
```
Comment (jverkoey) [inline CHANGELOG.md]: Typo?

**Comment by jverkoey** [comment]
At this point you may want to regenerate the changelog's automated release notes.

File: CHANGELOG.md (hunk 1 above)
Comment (romoore) [reply CHANGELOG.md]: Yep, thanks!

**Comment by romoore** [comment]
Why would I regenerate the release notes?  Did you find a specific error in the CHANGELOG?

**Comment by jverkoey** [comment]
Just the auto-generated portion. Otherwise the api diffs and listed commits may be out of date.

**Comment by romoore** [comment]
I'm not sure I understand - the original CHANGELOG was generated by release cut - should I recut the release?  I added the API Diffs recently (see the commit history), and I reviewed the Component Changes to make them more accurately reflect the release. Should I not modify those portions?

**Comment by jverkoey** [comment]
Run the release tool’s apidiff and notes commands.

**Comment by jverkoey** [comment]
Paste the results back in on top of the existing autogenerated notes and apidiff.

**Comment by jverkoey** [comment]
In place of. Not on top of. Replace the old notes and apidiff with the new ones.

**Comment by jverkoey** [comment]
Any hand-written notes can still live above the auto-generated portions. Make sure they’re accurate.

**Pull request merged**